apps/i18n/common/en_us.json

```diff
@@ -1411,6 +1411,8 @@
   "password": "Password",
   "passwordConfirmation": "Password confirmation",
   "passwordTooShort": "Password too short (minimum is 6 characters)",
+  "passwordUpdateFailed": "Password update failed. Please try again.",
+  "passwordsMustBeSixChars": "Passwords must be six (6) characters or longer.",
```

Comment:
Maybe out of scope for this change, but I wonder if we should make the password length in these strings configurable in case the minimum password length is ever updated. Probably not very likely though...

Reply:
The length restrictions are configured on the back-end so we'd need to pass the restrictions which is definitely possible, but yeah I think maybe out of scope for this change right now